### PR TITLE
Create home page for module admin

### PR DESCRIPTION
**Pull request by VoTaiTri**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -15,4 +15,8 @@ def sortable(column, title = nil)
     direction = (column == sort_column && sort_direction == "asc") ? "desc" : "asc"
     link_to title, params.merge(sort: column, direction: direction, page: nil), {class: css_class}
   end
+
+  def logged_in_admin?
+    !current_user.nil? && current_user.role == "admin"
```
Comment (NguyenHongViet) [inline app/helpers/application_helper.rb]: Dùng user_signed_in?

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
-    <%= link_to "Book Reviewing System", root_path, id: "logo" %>
+    <%= link_to_if(logged_in_admin?, "Admin Brs_10", admin_path, id: "logo"){} %>
+    <%= link_to_if(!logged_in_admin?, "Book Reviewing System", root_path, id: "logo"){} %>
```
Comment (NguyenHongViet) [inline app/views/layouts/_header.html.erb]: Dùng link_to_unless

**Comment by khuongnt1688** [comment]
ok

File: app/controllers/admin_controller.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class AdminController < ApplicationController
+  before_filter :authenticate_user!
+  before_filter :authorized_admin?
+
+  private
+  def authorized_admin?
+    unless current_user.role == "admin"
```
Comment (dieunb) [inline app/controllers/admin_controller.rb]: write a instance method `is_admin?` in User model

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root  'static_pages#home'
+  get 'admin' => 'admin/static_pages#home'
```
Comment (dieunb) [inline config/routes.rb]: remove

File: config/routes.rb [hunk 5]
```diff
@@ -12,6 +14,10 @@
     resources :comments
   end
 
+  namespace :admin do
+
```
Comment (dieunb) [inline config/routes.rb]: add root at here

**Pull request merged**